### DISCUSSION TITLE
ZEKE activation

### DISCUSSION THREAD

**Discussion by Laguna8899** [Security-Onion-Solutions/securityonion#9521] [Unsupported Versions]
Hello everyone,
i need to activate ZEKE on security onion on a machine where it has been deactivated.
How can I activate the zeke module? thank you

**Comment by cm-ops** [comment]
Would you provide a little more information on you grid please? Let me know what the following command returns `salt-call pillar.get global:mdengine`

**Reply by Laguna8899** [reply]
" Local: SURICATA" thank you for help

**Reply by cm-ops** [reply]
On the manager, edit the global pillar file `/opt/so/saltstack/local/pillar/global.sls` and change the mdengine variable from `SURICATA` to `ZEEK`. Then run a highstate on the node running sensor services.

**Reply by Laguna8899** [reply]
I need to collect the metadata so MDENGINE with SURICATA still keeping ZEEK enabled. how can I do?

**Comment by cm-ops** [comment]
For network metadata it is `Zeek` OR `Suricata` https://docs.securityonion.net/en/latest/suricata.html?highlight=mdengine#metadata You would still be using `Suricata` for signature based IDS.

**Reply by Laguna8899** [reply]
Thanks for the reply, I understand.
But if I want to use SURICATA for METADATA, zeek gets disabled. How can I keep zeek ENABLED while using SURICATA for METADATA?
thank a lot for help

**Reply by cm-ops** [reply]
I'll reiterate my above. For network metadata it is either `Zeek` OR `Suricata`, not both. You will still be running `Suricata` for NIDS (alerts).

**Reply by Laguna8899** [reply]
what if i want to zeek for nids (alerts) and suricata for metadata? how can I do?

**Reply by cm-ops** [reply]
You cannot. I would suggest you check out https://docs.securityonion.net/en/latest/zeek.html and https://docs.securityonion.net/en/latest/suricata.html
